CONTRIBUTING.md

```diff
@@ -836,15 +836,15 @@ depending on how large or controversial the changes to these components
 are. It is also admissible to have an assignee who is not a maintainer
 of any of the affected components, in case relevant maintainers are not
 available, and as long as the assignee is a member of the
-**@coq/pushers** team and is able to understand the changes in the PR.
+**@rocq-prover/pushers** team and is able to understand the changes in the PR.
 
 *If you have already frequently contributed to a component, we would
 be happy to have you join one of the maintainer teams.* See the
 [section below](#joining--leaving-maintainer-teams) on joining /
 leaving maintainer teams.
 
-The complete list of maintainer teams is available [here][coq-pushers]
-(link only accessible to people who are already members of the Coq
+The complete list of maintainer teams is available [here][rocq-pushers]
+(link only accessible to people who are already members of the Rocq
 organization, because of a limitation of GitHub).
```

Comment:
It is also reflected on the website now: https://rocq-prover.org/rocq-team
Using the infrastructure documented here: https://github.com/rocq-prover/rocq-prover.org/tree/main/data#github-teams

Suggestion to advertise this:
```suggestion
Prover organization, however, this list is also used to [generate](https://github.com/rocq-prover/rocq-prover.org/tree/main/data#github-teams)
the various [Rocq team][rocq-team] subpages on the website).
```